app/components/frame-viewer.cjsx

```diff
@@ -26,13 +26,31 @@ module.exports = React.createClass
     playing: false
     playbackRate: 1
     frameDimensions: {}
+    viewBoxDimensions: {
+      x: 0,
+      y: 0,
+      width: 0
+      height: 0
+    }
+    panEnabled: false
+    zooming: false
+    zoomingTimeoutId: 0
```

Comment:
Ok. In that case, you just need to change it to `zoomTimeoutId` when you assign it later.